.github/workflows/docker.yml

```diff
@@ -1,30 +1,50 @@
-name: 'Build bespoke Firedrake Docker container'
+name: Build bespoke Firedrake Docker container
 
 on:
+
+  # Build the Docker container whenever commits are pushed to an open PR
+  pull_request:
```

Comment:
Do we not need to have
```
on:
  push:
    branches:
      - main
  ```
  
  here as well?

Reply:
Not really, no. The `pull_request:` says we want the workflow to run whenever someone pushes a commit to the PR whereas this says we want it to run when the PR gets merged into `main`. It doesn't seem necessary for this workflow because it relates to a single file that's unlikely to have changed. (I'm trying to reduce Actions time usage where possible.)

Reply:
Alright, thanks :) on goalie and animate we run the workflow on both the pull request and main branch push, so I thought you might want the same here. But makes sense what you say!